.gitignore

```diff
@@ -47,3 +47,6 @@ flaskbotrc
 
 # mypy
 .mypy_cache
+
```

Comment:
I would remove this part of the commit; this file could be added to the .gitignore, but it would be better to discuss elsewhere and place into a separate PR.